src/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './App.css';
+import Calculator from './Component/Calculator';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- I would advise you to solve this linter Manually. Linters are tools that you use to debug your code. They scan your scripts for common issues and errors.